packages/local_auth/local_auth_android/android/src/main/java/io/flutter/plugins/localauth/AuthenticationHelper.java

```diff
@@ -74,7 +76,7 @@ interface AuthCompletionHandler {
       FragmentActivity activity,
       MethodCall call,
       AuthCompletionHandler completionHandler,
-      boolean allowCredentials) {
+      ArrayList<String> authenticationMethods) {
```

Comment:
We should avoid using string-based APIs internally; we have to use them for platform channels, but in other scenarios we should stick to things that would provide compile-time correctness checks.

Stepping back a bit though, do we need this change at all? It looks like this is passing in the enrolled authentication, but I wouldn't expect that we would need to restrict by that, vs. just letting the system pick what's available. Why not keep the bool, and then use it to decide whether to add `DEVICE_CREDENTIAL` to the default of `BIOMETRIC_WEAK | BIOMETRIC_STRONG`?